test/core/security/credentials_test.cc

```diff
@@ -710,6 +717,36 @@ static void test_refresh_token_creds_success(void) {
   grpc_httpcli_set_override(nullptr, nullptr);
 }
 
+static void test_refresh_token_with_quota_project_id_creds_success(void) {
+  grpc_core::ExecCtx exec_ctx;
+  expected_md emd[] = {
+      {"authorization", "Bearer ya29.AHES6ZRN3-HlhAPya30GnW_bHSb_"},
+      {"x-goog-user-project", "my-quota-project-id"}};
+  grpc_auth_metadata_context auth_md_ctx = {test_service_url, test_method,
+                                            nullptr, nullptr};
+  grpc_call_credentials* creds = grpc_google_refresh_token_credentials_create(
+      test_refresh_token_with_quota_project_id_str, nullptr);
+
+  /* First request: http put should be called. */
```

Comment:
nit: this comment is a little confusing, is it accurate? I don't think PUT requests are ever performed, in any case, if I'm correct?

Reply:
Good catch, changed to post.